frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetail.tsx

```diff
@@ -77,9 +78,15 @@ const mapDispatchToProps = (dispatch: any) => ({
   viewNextObjectDetail: () => dispatch(viewNextObjectDetail()),
   closeObjectDetail: () => dispatch(closeObjectDetail()),
 });
+type MapDispatchProps = ReturnType<typeof mapDispatchToProps>;
+
+type OwnProps = Omit<
+  ObjectDetailProps,
+  keyof MapStateProps | keyof MapDispatchProps
+>;
 
 // eslint-disable-next-line import/no-default-export -- deprecated usage
 export default connect(
   mapStateToProps,
   mapDispatchToProps,
-)(ObjectDetailWrapper);
+)(ObjectDetailWrapper) as unknown as React.ComponentType<OwnProps>;
```

Comment:
Fix the type so that TypeScript doesn't complain when I use the component in `frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetail.unit.spec.tsx`. Previously, it would complain I needed to pass props that would be passed automatically from `mapStateToProps` and `mapDispatchToProps`.